process.env

```diff
@@ -0,0 +1 @@
+HOST=http://localhost:8005
```

Comment:
can we go with Lukas started in https://github.com/RHEnVision/provisioning-backend/pull/54 (changing ports for all the other apps we depend on, but keep running on 8000 locally for our app?)